backend/src/controller/layers.rs

```diff
@@ -22,10 +19,9 @@ use crate::{model::dto::NewLayerDto, service::layer};
     params(
         ("map_id" = i32, Path, description = "The id of the map the layer is on"),
         LayerSearchParameters,
-        PageParameters
     ),
     responses(
-        (status = 200, description = "Search layers", body = PageLayerDto)
+        (status = 200, description = "Search layers", body = VecLayerDto)
```

Comment:
The layer DTO is quite small, so there is no significant performance increase in using a page there. It would make the layer handling code in the frontend a lot more difficult.

Reply:
I agree, there is no need to paginate the layers. Also, alternatives are hidden inside a collapsable and if we are concerned about the amount of user created layer on a map, we can lazy load just the alternative layers of the layer type the user wants to uncollapse.